CHANGELOG.md

```diff
@@ -1,3 +1,111 @@
+<a name="2.0.0-beta.4"></a>
+# [2.0.0-beta.4 unobtainium-sunglasses](https://github.com/angular/material2/compare/g3_v2_0...2.0.0-beta.4) (2017-05-12)
+
+### Breaking Changes
+
+* The `MdIconModule` no longer exports `HttpModule`, so you'll need to import it if you're using it elsewhere in your app.
+* The `forRoot` method on our modules has been removed. You should import our modules directly without this method. 
+
+### Bug Fixes
```

Comment:
Should add a section above bug fixes that's "Highlights" which mentions the datepicker, ability to turn off sanity checks, and anything else you think is noteworthy.